modules/402-ingress-nginx/hooks/safe_daemonset_update_test.go

```diff
@@ -17,607 +17,289 @@ limitations under the License.
+			pod := f.KubernetesResource("Pod", "d8-ingress-nginx", "controller-test-bw8sc")
+			Expect(pod.Field("metadata.labels.ingress\\.deckhouse\\.io/block-deleting").Exists()).To(BeTrue())
 		})
 	})
 
-	Context("daemonset proxy-main-failover update scheduled", func() {
+	Context("Checksums are not equal", func() {
 		BeforeEach(func() {
-			dsProxyMainFailover.Generation = 2
-			dsProxyMainFailover.Status.UpdatedNumberScheduled = 1
-			pod2ProxyMainFailover.Labels["pod-template-generation"] = "2"
-
-			dsControllerMainYAML, _ := yaml.Marshal(&dsControllerMain)
-			dsProxyMainFailoverYAML, _ := yaml.Marshal(&dsProxyMainFailover)
-			dsControllerMainFailoverYAML, _ := yaml.Marshal(&dsControllerMainFailover)
-			pod1ControllerMainYAML, _ := yaml.Marshal(&pod1ControllerMain)
-			pod2ControllerMainYAML, _ := yaml.Marshal(&pod2ControllerMain)
-			pod1ProxyMainFailoverYAML, _ := yaml.Marshal(&pod1ProxyMainFailover)
-			pod2ProxyMainFailoverYAML, _ := yaml.Marshal(&pod2ProxyMainFailover)
-			pod1ControllerMainFailoverYAML, _ := yaml.Marshal(&pod1ControllerMainFailover)
-			pod2ControllerMainFailoverYAML, _ := yaml.Marshal(&pod2ControllerMainFailover)
-
-			clusterState := strings.Join([]string{string(dsControllerMainYAML), string(dsProxyMainFailoverYAML), string(dsControllerMainFailoverYAML), string(pod1ControllerMainYAML), string(pod2ControllerMainYAML), string(pod1ProxyMainFailoverYAML), string(pod2ProxyMainFailoverYAML), string(pod1ControllerMainFailoverYAML), string(pod2ControllerMainFailoverYAML)}, "---\n")
-
-			f.BindingContexts.Set(f.KubeStateSet(clusterState))
-			f.BindingContexts.Set(f.GenerateAfterHelmContext())
-			_, _ = f.KubeClient().AppsV1().DaemonSets("d8-ingress-nginx").Create(context.TODO(), dsControllerMain, metav1.CreateOptions{})
-			_, _ = f.KubeClient().AppsV1().DaemonSets("d8-ingress-nginx").Create(context.TODO(), dsProxyMainFailover, metav1.CreateOptions{})
-			_, _ = f.KubeClient().AppsV1().DaemonSets("d8-ingress-nginx").Create(context.TODO(), dsControllerMainFailover, metav1.CreateOptions{})
-			_, _ = f.KubeClient().CoreV1().Pods("d8-ingress-nginx").Create(context.TODO(), pod1ControllerMain, metav1.CreateOptions{})
-			_, _ = f.KubeClient().CoreV1().Pods("d8-ingress-nginx").Create(context.TODO(), pod2ControllerMain, metav1.CreateOptions{})
-			_, _ = f.KubeClient().CoreV1().Pods("d8-ingress-nginx").Create(context.TODO(), pod1ProxyMainFailover, metav1.CreateOptions{})
-			_, _ = f.KubeClient().CoreV1().Pods("d8-ingress-nginx").Create(context.TODO(), pod2ProxyMainFailover, metav1.CreateOptions{})
-			_, _ = f.KubeClient().CoreV1().Pods("d8-ingress-nginx").Create(context.TODO(), pod1ControllerMainFailover, metav1.CreateOptions{})
-			_, _ = f.KubeClient().CoreV1().Pods("d8-ingress-nginx").Create(context.TODO(), pod2ControllerMainFailover, metav1.CreateOptions{})
-
-			f.RunHook()
-		})
-
-		It("pod proxy-main-failover-1 must be deleted", func() {
```

Comment:
Why are tests deleted?

Reply:
They were changed with respect of a new logic